app/utils/translationStrings.js

```diff
@@ -0,0 +1,1018 @@
+const translations = {
```

Comment:
`ember-i18n flattens the json`, and in some cases I need access to an entire dictionary (e.g. the qsort items). When refactoring these components into exp-addons, I just included the json I needed in the files directly so I won't need this file.